high_level/profile/profile_creation.md

```diff
@@ -0,0 +1,112 @@
+# Profile Creation - The technical part
+
+<!-- toc -->
+
+## What is a profile?
+
+All of the changes a user makes in Firefox, like the home page, what toolbars you use, installed addons, saved passwords and your bookmarks, are all stored in a special folder, called a profile.
+Telemetry stores archived and pending pings in the profile directory as well as metadata like the client ID.
+
+Every run of Firefox needs a profile. However a single installation can use multiple profiles for different runs.
+The profile folder is stored in a separate place from the Firefox program so that, if something ever goes wrong with Firefox, the profile information will still be there.
+
+Firefox also comes with a profile manager, a different run mode to create, migrate and delete the profiles.
+
+## Profile Behaviors
+
+In order to understand the behavior of users and base analysis on things like the profile creation date,
+it is essential to understand how a profile is created and identified by the browser.
+
+A profile can also be reset or deleted by the user.
+The user can opt-out of or opt-in to sending Telemetry data.
+
+These different cases are described in more detail in the following sections.
+
+### Profile Creation
+
+#### First use
+
+When Firefox is opened for the first time after a fresh install, without any prior Firefox profile on disk, it will create a new profile.
+Firefox chooses a random name, create the profile directory and marks the new profile as default for subsequent starts of Firefox.
+
+#### Default or Profile Manager creation
+
+On subsequent starts, Firefox tries to find the profile directory and load the profile data (bookmarks, addons, etc.).
+If not otherwise specified it uses the default (or last used) profile, finds the profile directory and starts loading files from there.
+
```

Comment:
The above part seems out of place with this section. It's talking about how FF looks for profiles and decides which profile to use, while this bigger section is about how profiles are created. 

might be a good idea to break out a separate section for what a "Default" profile is and it's role